### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abhatikar**
Fixed broken links to pre-trained model and demo application

### CODE REVIEW

**Review by raymondlo84 — COMMENTED** [review]
https://docs.openvinotoolkit.org/2020.4/usergroup13.html 

I think it should be 13... Sorry I think I provided you the wrong link in the first place.

File: pytorch_toolkit/action_recognition/README.md [hunk 1]
```diff
@@ -191,4 +191,4 @@ mo.py --input_model resnet34_vtn.onnx --input_shape '[1,16,3,224,224]'
 
 ## Demo
 
-You can try your models after converting them to the OpenVINO™ format or a [pretrained model from OpenVINO™](https://docs.openvinotoolkit.org/latest/usergroup10.html) using the [demo application from OpenVINO™ toolkit](https://docs.openvinotoolkit.org/latest/_demos_python_demos_action_recognition_README.html)
+You can try your models after converting them to the OpenVINO™ format or a [pretrained model from OpenVINO™](https://docs.openvinotoolkit.org/2020.4/usergroup10.html) using the [demo application from OpenVINO™ toolkit](https://docs.openvinotoolkit.org/2020.4/_demos_python_demos_action_recognition_README.html)
```
Comment (andrew-zaytsev) [inline pytorch_toolkit/action_recognition/README.md:194]: Are the links supposed to point to 2020.4 version?
To refer to the latest version, please use the following links:
https://docs.openvinotoolkit.org/latest/text_detection.html
https://docs.openvinotoolkit.org/latest/omz_demos_python_demos_action_recognition_README.html

**Review by andrew-zaytsev — CHANGES_REQUESTED** [review]

File: pytorch_toolkit/action_recognition/README.md (hunk 1 above)
Comment (raymondlo84) [reply pytorch_toolkit/action_recognition/README.md]: I see. 

So the 
https://docs.openvinotoolkit.org/2020.4/usergroup13.html 
is actually 
https://docs.openvinotoolkit.org/latest/text_detection.html

and 

https://docs.openvinotoolkit.org/2020.4/_demos_python_demos_action_recognition_README.html
is 
https://docs.openvinotoolkit.org/latest/omz_demos_python_demos_action_recognition_README.html

Sorry for the confusion. @abhatikar

**Comment by raymondlo84** [comment]
Wow! It's done =D!! Thank you everybody!

**Pull request closed without merging**